pages/doc/csp_invite-AoA-users_tutorial.md

```diff
@@ -136,29 +145,27 @@ Custom roles work only in combination with service roles. The Operations for App
 
 ### Step 2: Assign the Roles and Invite the User
 
-Let's assign **Organization Administrator** as a mandatory organization role, then assign the **Viewer** service role to one tenant and the **Ingestion Policies** role to another tenant. After that we will assign the custom role. 
+Let's assign **Organization Administrator** as a mandatory organization role, then assign the **Viewer** service role for one tenant and the **Ingestion Policies** service role for another tenant. After that we will assign the custom role and it will apply to the two tenants for which the user has service roles.
```

Comment:
After that, we assign the custom role, and it applies to the two tenants for which the user has service roles.